publication-commons/build.gradle

```diff
@@ -1,12 +1,33 @@
+repositories {
+    maven { url "https://s3.eu-central-1.amazonaws.com/dynamodb-local-frankfurt/release" }
+}
+
 def jacksonVersion = '2.10.2'
 
 dependencies {
     implementation group: 'com.github.BIBSYSDEV', name: 'nva-datamodel-java', version: 'v0.3.0'
```

Comment:
I could probably structure dependencies better in this project, open for suggestions!